src/Controller/CRUDController.php

```diff
@@ -1151,6 +1155,7 @@ protected function configure()
             ));
         }
 
+        // NEXT_MAJOR: remove this if. It is unnesessery becouse template registry is get directly from admin.
         $this->templateRegistry = $this->container->get(sprintf('%s.template_registry', $this->admin->getCode()));
         if (!$this->templateRegistry instanceof TemplateRegistryInterface) {
             throw new \RuntimeException(sprintf(
```

Comment:
This code will be unnesessery becouse `admin` will be configure correct (with template registry) before user call it in any controller.